known_issues.md

```diff
@@ -0,0 +1,876 @@
+
+###### AndroidManifestTemplate.bt ######
```

Comment:
Let's make a new issue to follow up on these older known issues. I'd still like to document them and make sure we eventually clear them out. I.e. remove the known_issues.md and make a github issue with its contents.